.github/pull_request_template.md

```diff
@@ -1,27 +1,10 @@
-## Description of the change
 
-> Description here
+I certify that:
 
-## Type of change
-- [ ] Bug fix (non-breaking change that fixes an issue)
-- [ ] New feature (non-breaking change that adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+* [ ] The readme and documentation has been updated or does not need to be updated
+* [ ] This change has been communicated with the team if needed
+* [ ] I have linked to the original issue
+* [ ] I have added tests to cover my changes
+* [ ] This PR does not need notes or I've added them below
+* [ ] This PR does not need extra testing from our release testing process or I've added a 'tests' section below
```

Comment:
Should an example test section be added that matches what Saitama is looking for in the `/release test` command?

Reply:
I think that could be the case but I think all too often people just delete that. (Including myself) this way you have to check a box either way Even if there is not a test.

Perhaps we should add that to the contributing.MD

Reply:
added here https://github.com/Plantiga/.github/pull/3/commits/22558c5dd963666788915a7203f1c55f933e271d